packages/driver/src/cypress/command_queue.ts

```diff
@@ -316,6 +316,12 @@ export class CommandQueue extends Queue<Command> {
       // store the previous timeout
       const prevTimeout = this.timeout()
 
+      // If we have created a timeout but are in an unstable state, clear the
+      // timeout in favor of the on load timeout already running.
+      if (!cy.state('isStable')) {
```

Comment:
Is there ever a scenario where we are unstable and we don't clear the timeout?

Reply:
If we are unstable we always want to rely on the onLoad timer except in the case cy.origin which also clears the timeout. afaik